semantic_release/cli/commands/changelog.py

```diff
@@ -62,10 +52,55 @@ def changelog(ctx: click.Context, release_tag: str | None = None) -> None:
             )
             ctx.exit(0)
 
-        changelog_text = render_default_changelog_file(env)
-        with open(str(changelog_file), "w+", encoding="utf-8") as f:
-            f.write(changelog_text)
+        with open(str(changelog_file), "a+", encoding="utf-8") as f:
+            f.seek(0)
+            existed_changelog_text = f.read()
+            f.truncate()
+            existed_changelog_sections = existed_changelog_text.split(CHANGELOG_PLACEHOLDER)
+            if len(existed_changelog_sections) > 1:
+                log.warning("Deprecation: Placeholder is detected, only new changes will be added.")
```

Comment:
:+1: for warning, but it would be good to include a link to [Migrating an Existing Changelog](https://python-semantic-release.readthedocs.io/en/latest/changelog_templates.html#changelog-templates-migrating-existing-changelog)